src/pages/blog/starlingx-debian-technology-preview-in-release-7.md

```diff
@@ -0,0 +1,510 @@
+---
+templateKey: blog-post
+title: Debian Technology Preview in Release 7
+author: Frank Miller and Eric Macdonald
+date: 2022-07-28T16:23:52.741Z
+category:
+  - label: News & Announcements
+    id: category-A7fnZYrE1
+---
+
+With support for the CentOS Distribution being discontinued, StarlingX will move
+to the Debian OS Distribution. Here's how to get ready.<!– more –>
+
+Debian is a well-established Linux Distribution supported by a large and mature
+open-source community and used by hundreds of commercial organizations,
+including Google. When fully transitioned to Debian, StarlingX will have full
+functional equivalence to the current CentOS-based versions of StarlingX.
+
+# Rollout
+
+The planned rollout for the transition to Debian is as follows:
+
+## StarlingX release 6.0 (RELEASED)
+
+* General Availability (GA) Release of CentOS7 StarlingX (for production
+  deployments)
+
+* Moved to 5.10 kernel, which will be used by the upcoming Debian-based release.
+
+## StarlingX release 7.0
+
+* StarlingX release 7.0 is a general Availability (GA) Release of CentOS7
+  StarlingX for production deployments. It will be the last release of a
+  CentOS7–based StarlingX.
+
+* StarlingX release 7.0 inherits the 5.10 version of the Linux kernel introduced
+  in StarlingX release 6.0.
+
+* StarlingX release 7.0 is also a technology Preview Release of Debian StarlingX
+  for evaluation purposes.
+
+* StarlingX release 7.0 release runs Debian Bullseye (11.3). It is limited in
+  scope to the AIO-SX configuration. Duplex, and standard configurations are not
+  available.
+
+See [Technology Preview Reduced Scope](#technology-preview-reduced-scope) for details.
+
+## Debian StarlingX General Availability
+
+An upcoming release will make Debian StarlingX generally available for
+production deployments.
+
+This upcoming release will run Debian Bullseye 11.3 or later with full
+functional equivalence to the CentOS-based StarlingX.
+
+## Planned in-service upgrade paths for StarlingX
```

Comment:
I suggest removing rows 56 to 65. While StarlingX code supports software upgrades, for Debian stx.7.0 is technology preview and there is no upgrade path yet to Debian.  we can provide upgrade info in the next release,